test/integration/cart_integration_test.rb

```diff
@@ -16,4 +24,19 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
       assert page.has_content?('1')
     end
   end
+
+  test 'an unauthorized user can add nine of the same item' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.